src/components/AppHeader.tsx

```diff
@@ -14,7 +14,7 @@ export const AppHeader = () => {
 					alt='Fischkapp logo'
 				/>
 			</a>
-			<p className='headerConteiner__cardInfo'>{`Cards:${count}`}</p>
+			<p className='headerConteiner__cardInfo'>{`Cards :${count}`}</p>
```

Comment:
**issue:** You don't have to use BEM when you have CSS modules configured. Just do:

```
<p className={styles.cardInfo}>{`Cards :${count}`}</p>
```

After that, your styles will be scoped to this specific component without the risk of overwriting from other component.

Reply:
Now it should be fine. I removed BEM and improved the code, and in the process, I learned something new :)